koswat/dike_reinforcements/input_profile/vertical_piping_solution/vps_input_profile_calculation.py

```diff
@@ -32,12 +34,12 @@ def __init__(self) -> None:
         self.scenario = None
 
     def build(self) -> VPSInputProfile:
-        self.reinforced_data = VPSInputProfile()
-
-        # Standard calculations
-        self.populate_profile(self.base_profile.input_data, self.scenario)
+        self.reinforced_data = self._get_reinforcement_profile(
```

Comment:
This what I meant at the protocol level, you do not need the property `reinforced_data`, as you are going to return it directly after this method, which is expected to the be the life cicle ending of `VpsInputProfileCalculation`.

Reply:
Note that I did not realize at this moment that this was declared earlier on at the base level. It seems we do not need to expose it anymore.

Reply:
Agree